src/images/searchicon.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" standalone="no"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg t="1615701135309" class="icon" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="1141" width="24" height="24" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><style type="text/css"></style></defs><path d="M908.7 909.4c-7.8 7.9-18.4 12.3-29.5 12.3-11.1 0-21.7-4.4-29.5-12.3L728.6 788.2c-10.8-10.5-15.1-26-11.3-40.5 3.8-14.5 15.2-25.9 29.7-29.7 14.5-3.8 30 0.5 40.5 11.3l121.2 121.2c7.9 7.8 12.3 18.4 12.3 29.5 0 11-4.4 21.6-12.3 29.4zM459.1 817c-98.6 0-174.7-27.4-239.3-92-70-68.7-117.9-166.6-117.9-265.2 0-197.3 159.9-357.3 357.3-357.3 197.3 0 357.3 159.9 357.3 357.3 0 98.7-40 188-104.7 252.6-67 67.1-157.9 104.7-252.7 104.6z m1.2-646.2c-103.7 0-199.4 55.3-251.3 145.1-51.8 89.8-51.8 200.4 0 290.1 51.9 89.7 147.6 145 251.3 145 160.2 0 290.1-129.9 290.1-290.2 0-160.2-129.9-290-290.1-290z m0 0" fill="#cdcdcd" p-id="1142"></path></svg>
```

Comment:
Why not put this SVG alongside the other SVGs in `src/utils/svg.js`? 

Or better, perhaps: maybe put those images here in this directory? In my experience svg-in-js is convenient, but it can be hard to grok if images are hiding in `.js` files in a codebase.